app/src/main/java/org/wikipedia/Constants.java

```diff
@@ -77,7 +77,13 @@
         ONBOARDING_DIALOG,
         FEED,
         NOTIFICATION,
-        APP_SHORTCUTS
+        APP_SHORTCUTS,
+        TOOLBAR,
+        WIDGET,
+        INTENT_SHARE,
+        INTENT_PROCESS_TEXT,
+        FEED_BAR,
+        VOICE
```

Comment:
Not sure but should these `Search` related invoke source can have names like `SEARCH_VOICE`?

Reply:
I believe the point of these patches is to abstract away the various invoke sources into general-purpose constants, usable from multiple contexts. Therefore in this case it might actually be beneficial to have more general names than specific ones.